PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+
+### Week 3, {DATES_WEEK_3}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| {DEV_3} & {DEV_1}     | [6. As a user, I want to be able to invite others to an existing shopping list.]({REPO_URL}/issues/6)                   |
+| {DEV_4} & {DEV_2}     | [7. As a user, I want to filter my shopping list to make it easier to locate an item in the list.]({REPO_URL}/issues/7) |
+
+| 📖 Learning modules                                    | Description |
+| ------------------------------------------------------ | ----------- |
+| [How to tell your story](https://youtu.be/Zhtd-DdTwSs) |             |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Demo issues #4 & #5 <br> - Retro <br> - This week's issue                                                                                                       |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue. |
+| Optional         | Office Hours                                                                                                                                                      |
+
+### Week 4, {DATES_WEEK_4}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                                      |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_1} & {DEV_2}     | [8. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works. ]({REPO_URL}/issues/8) |
+| {DEV_3} & {DEV_4}     | [9. As a user, I want to mark an item on my shopping list as purchased so the app can learn how often I buy different items.]({REPO_URL}/issues/9)            |
+
+| 📖 Learning modules                                                                           | Description |
+| --------------------------------------------------------------------------------------------- | ----------- |
+| [What to include on your Linkedin profile](./learning-modules/what-to-include-on-linkedin.md) |             |
```

Comment:
**Possible Description:** In this module, you'll learn the steps you need to take to prepare your LinkedIn for a mentor review. 

_Bulleted description, if needed:_
You'll learn how:

- mentors will be grading your profile
- to update your photo, title, background, About Section, Featured Section
- tips for your Experience, Education, Certifications, and Volunteer sections